src/components/Home/Home.tsx

```diff
@@ -13,11 +13,20 @@ const Home: React.FC = () => {
 			</section>
 			<nav className="cta-buttons">
 				<OpenMenuContext.Consumer>
-					{({ toggleMenu }) => (
-						<button className="cta-button" onClick={toggleMenu}>Find Applicants</button>
+					{({isOpen, toggleMenu }) => (
+						<>
+							<button className="cta-button" onClick={toggleMenu}>Find Applicants</button>
+							<Link 
+								to="/" 
+								onClick={() => {
```

Comment:
I think that this onClick will need to be moved to the Find Applicants button as that button is no longer opening the menu.

Reply:
Good catch!

Reply:
Nice! Got it in before the merge, that shouldn't be a problem anymore.